include/openspace/interaction/joystickcamerastates.h

```diff
@@ -60,7 +61,15 @@ class JoystickCameraStates : public CameraInteractionStates {
         AxisInvert invert = AxisInvert::No;
         AxisNormalize normalize = AxisNormalize::No;
 
+        // The axis values can either go back to 0 when the joystick is released or it can
+        // stay at the value it was before the joystick was released.
+        // The latter is called a static axis, when the values don't go back to 0.
+        bool isStatic = false;
```

Comment:
I'd imagine that a "static axis" is an axis that doesn't change..  Maybe a different term might be better?   Not really sure what a good term would be.. .  "integrated axis"?

Reply:
I agree that it is not a perfect term. How about sticky? It feels more intuitive that it can be moved but it will stick to the previous value when let go.

Reply:
I can live with "sticky" :)